TESTING.md

```diff
@@ -249,6 +249,39 @@ cargo make watch-test
 
 **Note:** For more information on publishing AMIs see [publishing](PUBLISHING.md).
 
+### vmware-k8s
+You need to [build](BUILDING.md) Bottlerocket and a TUF repo to test VMware variant.
+Either `Infra.toml` or your environment need to be configured.
+Make sure to set: GOVC_URL, GOVC_USERNAME, GOVC_PASSWORD, GOVC_DATACENTER, GOVC_DATASTORE, GOVC_NETWORK, GOVC_RESOURCE_POOL, GOVC_FOLDER.
+If using `Infra.toml` the datacenter specified in `Test.toml` will be used, or the first one in the datacenters list of `Infra.toml` will be used if no datacenter is specified.
+VMware testing also requires a `control-plane-endpoint` to be set in `Test.toml` for vmware cluster creation.
+Change the commands below to the desired `vmware-k8s` variant:
+
+```shell
+cargo make \
+  -e BUILDSYS_VARIANT="vmware-k8s-1.23" \
+  -e BUILDSYS_ARCH="x86_64" \
+  build
+
+cargo make \
+  -e BUILDSYS_VARIANT="vmware-k8s-1.23" \
+  -e BUILDSYS_ARCH="x86_64" \
+  repo
+```
+
+Sync TUF repos with the vmware metadata and targets.
```

Comment:
```suggestion
Sync TUF repos containing the VMware variant's metadata and targets.
```

Reply:
Is it clear what "sync TUF repos means"? What we really mean is, put the TUF repo in a place that the testsys resource agents can access via HTTP. And I guess this also needs to match the URLS in Infra.toml, right? Maybe say a little more here about what is happening for the uninitiated.